salt/utils/win_runas.py

```diff
@@ -392,10 +392,37 @@ def runas_system(cmd, username, password):
     return ret
 
 
+def get_sid_from_name(name):
```

Comment:
To refactor this, just import `salt.utils` at the top of this file and then call `salt.utils.get_sid_from_name` where neccesary.